tonic/models/vic/grid_params.py

```diff
@@ -602,6 +732,71 @@ def __init__(self, organic_fract=False, spatial_frost=False,
                                'basin_depth': 'm',
                                'basin_area': 'fraction'}
 
+        # State Variables
+        self.state = {'gridcell': 'N/A',
+                      'Soil_moisture': 'mm',
+                      'Soil_ice': 'mm',
+                      'Canopy_water': 'mm',
+                      'Snow_age': 'time steps',
+                      'Snow_melt_state': '0 or 1',
+                      'Snow_coverage': 'fraction',
+                      'Snow_water_equivalent': 'mm',
+                      'Snow_surf_temp': 'deg C',
```

Comment:
This should technically be in Kelvin to be CF complaint but I don't think we should do that here. For now, let's just go with `'C'`.